lib/proxy/log.rb

```diff
@@ -20,6 +20,30 @@ def logger
   end
 
   class LoggerFactory
+    class LogFormatter < ::Logger::Formatter
+      Format = "%s, [%s%s] %5s -- %s: %s\n".freeze
+
+      def call(severity, time, progname, msg)
+        Format % [severity[0..0], format_datetime(time), request_id[0..5], severity, progname, msg2str(msg)]
```

Comment:
Could you use eight characters of the request ID? Mostly to match what Foreman currently logs - if the same ID is used then it'll be identical in both log files.